### PR TITLE
Feature/approve reject borrowers

### LINKED ISSUE #467: [MicroCredit] add approve/reject on borrowers list
dependent on https://github.com/impactMarket/backend/issues/598

### PR DESCRIPTION
**Pull request by marcusats**
This PR fixes #467 

## Changes

- 

## Tests

- [ ] Tested on mobile
- [ ] Tested on dektop

This PR should be manually tested by @obernardovieira @PauloSousapt @hugomarquesdev , included on reviewers.

### CODE REVIEW

File: src/views/MicrocreditManager/RequestTab.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            title: 'Approved',
+            number: 3,
+            onClick: () => console.log('Approved')
+        },
+        {
+            title: 'Rejected',
+            number: 2,
+            onClick: () => console.log('Rejected')
+        }
+    ];
```
Comment (hugomarquesdev) [inline src/views/MicrocreditManager/RequestTab.tsx:92]: are the console.logs needed?

**Review by hugomarquesdev — CHANGES_REQUESTED** [review]

**Comment by hugomarquesdev** [comment]
also, send me a message with the translations you need. I'll add them on prismic and then I'll send you the keys

File: src/views/MicrocreditManager/RequestTab.tsx (hunk 1 above)
Comment (marcusats) [reply src/views/MicrocreditManager/RequestTab.tsx:92]: I add it just for testing. I can delete that

**Comment by hugomarquesdev** [comment]
I see that there are still some texts written directly on the code. Can you send me a message in slack with the translations you need so I can add them on Prismic?

**Comment by obernardovieira** [comment]
closing in favor of https://github.com/impactMarket/app/pull/492

**Pull request closed without merging**